recipes/mkl-service/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "2.0.1" %}
+{% set buildnumber = 0 %}
+
+package:
+    name: mkl-service
+    version: {{ version }}
+
+source:
+    url: https://github.com/IntelPython/mkl-service/archive/v{{version}}.tar.gz
+    sha256: 1ae3032445f6af07071118e9af19b8cd05c72235fe47f4d214035dfdcd8aabfa
+
+build:
+    number: {{ buildnumber }}
+
+requirements:
+    build:
+      - {{ compiler('c') }}
+    host:
+      - python
+      - setuptools
+      - mkl-devel
+      - cython
+      - numpy 1.11.*
+      - six
```

Comment:
I don't think `six` is needed in the `host` requirements?
```suggestion
```
(`host` should only contain dependencies of the `setup.py` script.)

Reply:
Thanks @chrisburr , I removed the line, as well as corrected `license` and added `license-family`. 

I would like to keep using jinja variables. I like the fact that all I need to change when updating the recipe is atop of the `meta.yaml`.